### ISSUE TITLE
Enforce use of Definition of Done

### ISSUE THREAD

**Issue by lkocman** [openSUSE/openSUSE-release-process#5]
This could be a continuation of the Issue #4  and is essentially blocked on it

I wrote a pre-commit hook for the gitlab/SLE to double check whether each new task has__Definition of done__ set.

There are ways how this can be tricked, and it causes problems with rebases, however works fine if your HEAD is up2date. 

[pre-receive.txt](https://github.com/openSUSE/openSUSE-release-process/files/4179948/pre-receive.txt)

SLE pre-commit-hook location
https://gitlab.suse.de/sle-prjmgr/release-management-checklist/blob/master/hooks/pre-receive

Documentation about the coomit hook: https://confluence.suse.com/display/projectmanagement/Git+hooks#Githooks-pre-commithook
